_config.yml

```diff
@@ -11,7 +11,9 @@ branch: master
 #   {{site.webdev}}/downloads
 
 dartlang: ""
+prev-dartlang: "https://v1-dartlang-org.firebaseapp.com"
```

Comment:
If we are still going with the naming convention that we had agreed to, then this should be named `prev-url` (or at least we should have both names) so that we end up with:

```yaml
url: https://dartlang-org-dev.firebaseapp.com # this is from line 5 above
prev-url: https://v1-dartlang-org.firebaseapp.com
```

But as I mentioned, this isn't critical, it can be changed later as needed.

Reply:
Oh, you mean literally `url`? I thought `url` was a stand-in for whatever we decided to call this current site. :)